internal/driver/glfw/window_desktop.go

```diff
@@ -677,6 +678,32 @@ func (w *window) charInput(viewport *glfw.Window, char rune) {
 	w.processCharInput(char)
 }
 
+var counterIme int
+
+// IME
```

Comment:
A comment should not be needed here. Where comments are added it usually implies naming is insufficient.